Earthfile

```diff
@@ -1,24 +1,38 @@
-FROM node:13.10.1-alpine3.11
-
-RUN npm install -g vsce
+VERSION 0.7
```

Comment:
0.7 isn't compatible with the `+secrets/earthly-technologies/ovsx/token` format.

From the v0.7.0 CHANGELOG:

> The Cloud-based secrets model is now project-based; it is not compatible with the older global secrets model. Earthfiles which are defined as VERSION 0.5 or VERSION 0.6 will continue to use the old global secrets namespace; however the earthly command line no longer supports accessing or modifying the global secrets. A new earthly secrets migrate command has been added to help transition the global-based secrets to the new project-based secrets.

We should create a new project (you might be able to do that via `earthly project create earthfile-grammar`, or I can do that for you if you don't have permission), then we'll need to define `PROJECT earthly-technologies/earthfile-grammar` after the `VERSION`, and will have to migrate the secrets over:

run `./scripts/install-all-versions.sh` in the earthly core repo to install older versions, then use a command like:

```
earthly-v0.6.30 secrets get -n /earthly-technologies/vsce/token | earthly-v0.7.13 secrets --org earthly-technologies --project earthfile-grammar set --stdin vsce-token
```

make sure you use the `-n` flag to avoid copying in the newline that `get` typically appends.

then finally update the RUN commands:

```
RUN -secret VSCE_TOKEN=vsce-token ...
```